src/components/typography/text/text.js

```diff
@@ -31,6 +33,7 @@ const Subheadline = props => {
         [styles[`${props.tone}`]]: props.tone,
         [styles.truncate]: props.truncate,
       })}
+      title={props.title}
```

Comment:
can you strip this out if it's not defined?

Reply:
Looks like it's not passed to DOM in case it's `undefined`. Behaviour you were observing happens if `title` is an empty string. So, I will add fallback to `undefined` if has a falsy value.